src/util/fs.js

```diff
@@ -13,6 +13,15 @@ const globModule = require('glob');
 const os = require('os');
 const path = require('path');
 
+export const constants =
+  typeof fs.constants !== 'undefined'
```

Comment:
Comment about Node 4 compatibility here would be great so in the future we don't get confused about this code.